### DISCUSSION TITLE
Show debug info only with blueprints and Lyra combine

### DISCUSSION THREAD

**Discussion by ZacharyDK** [Sixze/ALS-Refactored#323] [Q&A]
1. Not sure how to get debug info to display. Shift + 0 does nothing.
2. Previous versions had a debug component. This has been refactored into AlsCharacter_Debug.cpp
3. There isn't anything that appears when type debug in the search bar in the content browser. 
4. Nothing obvious in blueprint. I feel shame. 

I am specifically looking to figure out how to debug overlay animations. I am looking to combine ALS w/ Lyra, so I can easily access everything I want to. I was able to successfully refactor the plugin as a game feature, and have ALSCharacter and ALsMovementComp inherit from Lyra's character and Lyra's char movement comp. Unfortunately, just setting up the inheritance breaks ALS. Sprint/Walk/Idle and legs just don't work on the Als character. There are a million places where things can go wrong. 

You can try for yourself by adding "AIModule","LyraGame","ModularGameplay" "GameplayAbilities" to ALS build.cs file. 

Additional dependencies in the uplugin file for ALS:: GameplayAbilities, AIModule

New Folder will be Plugins/GameFeatures 
.uplugin file also needs. 
"ExplicitlyLoaded": true,
"BuiltInInitialFeatureState": "Active",

ALS extra build.cs needs GameplayAbilities","LyraGame","ModularGameplay in public dependencies.

**Comment by ameaninglessname** [comment]
Have you copied [these content](https://github.com/Sixze/ALS-Refactored/blob/main/Config/DefaultInput.ini) to your `DefaultInput.ini`?

**Comment by ZacharyDK** [comment]
Those are already in the default ini. I'll have a look at the console commands
